### DISCUSSION TITLE
Alert queries fail with "input data must be wide series but got type not (input refid)"

### DISCUSSION THREAD

**Discussion by ssantrosyan** [grafana/grafana#61536] [Alerting]
Hi, I am creating an alerting rule by Loki query and got this error. Using Grafana 9.2.6 version. how can I solve this issue?

![image](https://user-images.githubusercontent.com/64489825/212643690-c1118529-09e9-4271-8f38-fd38b63d8d50.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
